### ISSUE TITLE
Current Google RSS URLs are deprecated

### ISSUE THREAD

**Issue by yan12125** [yan12125/chrome_newtab#3]
For example on https://news.google.com.tw/news?pz=1&cf=all&ned=tw&hl=zh-TW&output=rss&topic=y:
```
This RSS feed URL is deprecated

This RSS feed URL is deprecated, please update. New URLs can be found in the footers at https://news.google.com/news
```

**Comment by yan12125** [comment]
Web page: https://news.google.com/news/headlines/section/topic/NATION.zh-TW_tw?ned=tw&hl=zh-TW&gl=TW
RSS Link: https://news.google.com/news/rss/headlines/section/topic/NATION.zh-TW_tw?ned=tw&hl=zh-TW&gl=TW

Query strings are necessary. Ex: title is wrong in https://news.google.com/news/rss/headlines/section/topic/NATION.zh-TW_tw

TODO:
* Support placeholders
* Replace old ``prefix`` approach with placeholders?

**Comment by yan12125** [comment]
https://news.google.com/news/rss/headlines/section/topic/WORLD.zh-TW_tw?ned=tw
https://news.google.com/news/rss/headlines/section/topic/NATION.zh-TW_tw?ned=tw
https://news.google.com/news/rss/headlines/section/topic/BUSINESS.zh-TW_tw?ned=tw
https://news.google.com/news/rss/headlines/section/topic/SCITECH.zh-TW_tw?ned=tw
https://news.google.com/news/rss/headlines/section/topic/SPORTS.zh-TW_tw?ned=tw
https://news.google.com/news/rss/headlines/section/topic/ENTERTAINMENT.zh-TW_tw?ned=tw
https://news.google.com/news/rss/headlines/section/topic/MAINLAND_CHINA.zh-TW_tw?ned=tw
https://news.google.com/news/rss/headlines/section/topic/HEALTH.zh-TW_tw?ned=tw

**Comment by yan12125** [comment]
Fixed in a8196d17a62af133094a2894bf501b34252784e0
